softserve/views.py

```diff
@@ -80,35 +72,57 @@ def dashboard():
 
 
 @app.route('/create_node', methods=['GET', 'POST'])
-#@organization_access_required('gluster')
+@organization_access_required('gluster')
 def get_node_data():
-    count = db.session.query(func.count(Vm.id)) \
-            .filter_by(state = 'ACTIVE').scalar()
-    if count >=5:
-        flash('Limit got over!')
-        return render_template('home.html')
+    if request.method == "POST":
+        counts = request.form['counts']
+        name = request.form['node_name']
+        hours_ = request.form['hours']
+        pubkey_ = request.form['pubkey']
+
+        # Validating the hours and node counts
+        if counts > 5 or hours_ > 4:
+            flash('Please enter the valid data')
+            logging.exception('User entered the invalid hours or counts value')
+            return render_template('form.html')
```

Comment:
You need to send `n` back to the template every time you call `render_template('form.html')`